src/routes/NonAuthLayout/Login/Login.js

```diff
@@ -51,7 +51,7 @@ export default class Login extends React.Component {
           internalLabel='Email'
           onChange={this.handleChange}
           styleName='field'
-          type='text'
+          type='email'
```

Comment:
a bit random here, but alright

Reply:
Sure. It's been on my list for a long time. Really bugs me when I'm testing mobile. Can remove it, but it's free 😅